corehq/apps/es/client.py

```diff
@@ -1091,40 +1091,31 @@ def bulk(self, actions, refresh=False, raise_errors=True):
             for action in chunk:
                 # apply the chunk to both indexes in parallel
                 payload.append(self.primary._render_bulk_action(action))
-                payload.append(self.secondary._render_bulk_action(action))
+                if action.is_delete:
+                    # Always create a tombstone on secondary index when delete delete is issued on primary
+                    if action.doc is None:
+                        doc_id = action.doc_id
+                    else:
+                        doc_id = self.from_python(action.doc)[0]
+                    payload.append(
+                        self.secondary._render_bulk_action(
+                            BulkActionItem.index(Tombstone(doc_id)),
```

Comment:
Is it OK to create this doc with the same ID as a doc that's being deleted in the same bulk request.

Reply:
I think so. They both are happening on different indexes, so it should be fine. Do you have some thoughts on where it can go wrong?

Reply:
>  They both are happening on different indexes

Ah right, I see you deleted the other operation for the secondary index.